cli/standalone/mod.rs

```diff
@@ -574,12 +574,14 @@ pub async fn run(
     false,
   );
 
+  // Initialize v8 once from the main thread.
   v8_set_flags(construct_v8_flags(&[], &metadata.v8_flags, vec![]));
+  deno_core::JsRuntime::init_platform(None);
```

Comment:
Seems like a bug that this was never initialized?

Reply:
Indeed